src/vs/platform/windows/electron-main/window.ts

```diff
@@ -1325,9 +1326,14 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 
 		const useCustomTrafficLightPosition = this.configurationService.getValue<boolean>(commandCenterSettingKey);
 		if (useCustomTrafficLightPosition) {
-			this._win.setTrafficLightPosition({ x: 7, y: 9 });
+			if (!this.defaultTrafficLightPosition) {
+				this.defaultTrafficLightPosition = this._win.getTrafficLightPosition(); // remember default to restore later
+			}
```

Comment:
@bpasero On first call this does return `{x:0, y:0}` which seems fishy and therefore I didn't persist the old value. Is the (0,0) a well-known value for default?